catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -274,8 +264,6 @@
                             quartz-jobs,
                             c3p0,
                             terracotta-toolkit-runtime-ee,
```

Comment:
So far, removing this causes iTest failure. Trying to research why within a decent time frame.